### DISCUSSION TITLE
"Synthesizing x/10 solutions" but it doesn't provide any results.

### DISCUSSION THREAD

**Discussion by gokaybiz** [community/community#60884] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

![img1](https://i.hizliresim.com/jr15ixj.png)
![img2](https://i.hizliresim.com/d2rrzkw.png)
![img3](https://i.hizliresim.com/1w8g0ap.png)

Each time it returns different sequence but same blank output. 
latest vscode on osx with latest copilot extension with student pack.

thanks in advance...

**Comment by gokaybiz** [comment]
problem solved, I removed github authentication and authorized again.

**Comment by gokaybiz** [comment]
Well, when I use copilot labs, normal copilot stops working, until I sign out and log in again...

**Comment by davecheney** [comment, marked as answer]
@gokaybiz we believe we have corrected the problem and deployed a fix. Could you please retry and let us know if this helped.
